### PR TITLE
Add script to make amq-online-install zip

### PR DESCRIPTION
**Pull request by k-wall**

### CODE REVIEW

File: make_install_zip.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Makes the amq-online-install zip
+
```
Comment (ctron) [inline make_install_zip.sh:3]: can you add `set -e`

File: make_install_zip.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+if [ ! "${TAG}" ] 
+then
+   1>&2 echo "Usage: $0 <tag>"
+   exit 1
+fi
+
+mkdir -p ${DIR}/target
+TARGET_ZIP=$DIR/target/amq-online-install-${TAG}.zip
+
+(cd templates; zip -qr ${TARGET_ZIP} .)
```
Comment (ctron) [inline make_install_zip.sh]: I think the `cd` should also make use of `$DIR`

**Review by ctron — CHANGES_REQUESTED** [review]

File: make_install_zip.sh (hunk 1 above)
Comment (k-wall) [reply make_install_zip.sh:3]: Done

File: make_install_zip.sh (hunk 2 above)
Comment (k-wall) [reply make_install_zip.sh]: Done too.

**Review by ctron — APPROVED** [review]

**Pull request merged**